app/constants/claim.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  livestockTypes: {
+    beef: 'beef',
+    dairy: 'dairy',
+    pigs: 'pigs',
+    sheep: 'sheep'
+  },
+  claimType: {
+    review: 'R',
+    endemics: 'E'
+  },
+  testResults: {
+    positive: 'positive',
+    negative: 'negative'
+  },
+  speciesNumbers: {
+    yes: 'yes',
+    no: 'no'
+  },
+  numberOfOralFluidSamplesLimit: 5,
+  numberOfAnimalsTestedLimit: {
+    beef: 5,
+    pigs: 30,
+    sheep: 10
+  }
```

Comment:
The word 'limit' to me implies that it is the MAX number allowed whereas it is actually the MIN. Can you rename to include the word 'minimum', 'threshold', or something like that?